dev-python/svg-path/svg-path-3.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="svg.path"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="SVG path objects and parser"
+HOMEPAGE="https://github.com/regebro/svg.path"
+SRC_URI="https://github.com/regebro/svg.path/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+python_test() {
+	esetup.py test || die
+}
+
+src_install() {
+	MODNAME="svg"
```

Comment:
Once again, why do you need this special variable?

Reply:
Dependencies expect to be able to load it via `from svg.path import`, I need to specify that the top level module name is `svg`.

Reply:
Still don't see why you need to make it a variable.

Reply:
ah yes, I finally understand what you mean :DD fixed